app/javascript/packages/document-capture/components/acuant-capture.tsx

```diff
@@ -225,7 +240,7 @@ function suspendFocusTrapForAnticipatedFocus(focusTrap) {
   }, 0);
 }
 
-export function getDecodedBase64ByteSize(data) {
+export function getDecodedBase64ByteSize(data: string | any[]) {
```

Comment:
Can we drop the `| any[]` here?

```suggestion
export function getDecodedBase64ByteSize(data: string) {
```